sec-scanners-config.yaml

```
@@ -1,6 +1,6 @@
 module-name: istio
 protecode:
-  - europe-docker.pkg.dev/kyma-project/prod/istio-manager:v20240925-42ba4909
+  - europe-docker.pkg.dev/kyma-project/prod/istio-manager:v20241004-842cc619
```

That image does not exist 🤔 
```suggestion
  - europe-docker.pkg.dev/kyma-project/prod/istio-manager:v20241004-67eb80cf
```